gym/wrappers/step_api_compatibility.py

```diff
@@ -11,37 +14,36 @@ class StepAPICompatibility(gym.Wrapper):
     New step API refers to step() method returning (observation, reward, terminated, truncated, info)
     (Refer to docs for details on the API change)
 
-    This wrapper is to be used to ease transition to new API and for backward compatibility.
-
     Args:
         env (gym.Env): the env to wrap. Can be in old or new API
-        new_step_api (bool): True to use env with new step API, False to use env with old step API. (False by default)
+        apply_step_compatibility (bool): Apply to convert environment to use new step API that returns two bools. (False by default)
```

Comment:
Rephrase, the "Apply" sounds weird. Probably would be best as something like "Whether or not to [...]"

Also now I think it defaults to True

Reply:
I think it should default to `False`. We **shouldn't** apply the compatibility wrapper by default